### PR TITLE
Add inplace options in the process function

### PR DESCRIPTION
**Pull request by sagars729**
- Add inplace options in CharPostprocessor and StructCharPostprocessor
- Add tests to check that the inplace options works
- https://github.com/capitalone/DataProfiler/issues/91

### CODE REVIEW

File: dataprofiler/labelers/data_processing.py [hunk 1]
```diff
@@ -926,7 +927,7 @@ def convert_to_NER_format(predictions, label_mapping, default_label,
         return output_result
 
     @staticmethod
-    def match_sentence_lengths(data, results, flatten_separator):
+    def match_sentence_lengths(data, results, flatten_separator, inplace=True):
         """
         Converts the results from the model into the same ragged data shapes as
         the original data.
```
Comment (lettergram) [inline dataprofiler/labelers/data_processing.py:933]: Need to add the parameter description.

File: dataprofiler/labelers/data_processing.py [hunk 2]
```diff
@@ -1328,7 +1332,7 @@ def help(cls):
         print(help_str)
 
     @staticmethod
-    def match_sentence_lengths(data, results, flatten_separator):
+    def match_sentence_lengths(data, results, flatten_separator, inplace=True):
```
Comment (lettergram) [inline dataprofiler/labelers/data_processing.py:1335]: Need to add the parameter description.

Kinda weird this function occurs twice

**Review by lettergram — CHANGES_REQUESTED** [review]

**Review by AnhTruong — COMMENTED** [review]
agree with Austin. Please add descriptions.

**Comment by sagars729** [comment]
@lettergram @AnhTruong added parameter descriptions

**Review by ChrisWallace2020 — APPROVED** [review]

**Review by lettergram — APPROVED** [review]

**Pull request merged**